.github/workflows/alpha_release.yml

```diff
@@ -0,0 +1,48 @@
+name: Create Alpha Release
+on:
+  push:
+    branches:
+      - main
+  
+jobs:
+  build:
+    name: Create Alpha Release
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Package Version
+      id: package-version
+      uses: martinbeentjes/npm-get-version-action@master
+    - uses: actions/setup-node@v1
+      with:
+        node-version: '10.x'
+    - name: Install
+      run: npm ci --no-optional
+    - name: Package
+      run: npm run vsce:package
+    - name: Tag commit
+      uses: tvdias/github-tagger@v0.0.1
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
+        tag: ${{ steps.package-version.outputs.current-version }}-alpha
+    - name: Create Release
+      id: create_release
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
+      with:
+        tag_name: ${{ steps.package-version.outputs.current-version }}-alpha
```

Comment:
we can release multiple alphas, it will modify the `updated_at` property which will be checked upon installation 
https://github.com/codota/tabnine-vscode/blob/b08cb1081517c15a50a7af4ad323829414e4f3c8/src/alphaInstaller.ts#L38

Reply:
actually, it will be much cleaner to add a timestamp and to check the asset update time. I've changed it, it will create a new  artifact for each push